src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs

```diff
@@ -190,10 +190,12 @@ private void SynchronizeSelectedItem()
 					{
 						pivotItem.Opacity = 0;
 						pivotItem.IsHitTestVisible = false;
+						pivotItem.Visibility = Visibility.Collapsed;
```

Comment:
Does it still need to set Opacity and IsHitTestVisible if it's setting visibility to Collapsed?

Reply:
1. Same [here](https://github.com/nventive/Uno/pull/77/files#diff-b075b6b5f6cb0c0eb8643ddfd5035f1bR197)
2. I'm not so involved, but setting the opacity and hit-test-visibility here might take away custom configuration (e.g. lower opacity) from the user and will reset it on selection synchronization.

Reply:
Fair enough, this is an extrapolation of the behavior coming from UWP, where the opacity seems to be changed as well, but this is not required for the current implementation.

Collapsed-only will be enough.